### ISSUE TITLE
Expecting Json data.  Check your Content-Type header

### ISSUE THREAD

**Issue by ChevelleNV** [nxgninnovations/Signifyd#1]
Get "Expecting Json data.  Check your Content-Type header" when i visit the Signfyd Dashboard. 

also you should package a real install of the plugin and also get it listed, it doesnt show up when searching new plugins in Wordpress, had to pull the folder out of the downloaded folder from git hub to get it to install. most non technical users wont know what to do if they have to fiddle with the files to upload it.

**Comment by dabodude** [comment]
Agreed... Didn't notice it was inside another folder until I read this... Thanks @ChevelleNV

**Comment by dabodude** [comment]
@ChevelleNV Have you managed to fix this issue? Found any other solution to integrate Signifyd to WooCommerce?

**Comment by ChevelleNV** [comment]
@dabodude No, never fixed the issue, but its only a problem on the dashboard page. It still runs the scoring on the orders for woo commerce and shows the score on the order details page. I just don't ever go to the dashboard page for the plugin

**Comment by ChevelleNV** [comment]
so I am assuming since there has been not even so much as a peep from the developer in months on this that this plugin has been abandoned...

**Comment by techrah** [comment]
Make sure the plugin sends a single IP address when creating a new case and not multiple IP addresses separated by a comma, for example.

**Comment by ChevelleNV** [comment]
and how am I supposed to do that?

**Comment by techrah** [comment]
First of all, this may not be the same issue you're having but because my web server was behind a reverse web proxy, multiple IP addresses were forwarded along in the final request to the actual web server that hosted WordPress and WooCommerce. My setup ensured that the very first IP address was the actual address of the requesting client so for me it was sufficient to simply extract the first one. I think this was enough for the plugin to start working for me.

So the solution to this is to edit the woocommerce-signifyd.php file and change the following line (around line 193) from:
`'browserIpAddress'  => get_post_meta( $post_id, '_customer_ip_address', true ),`
to:
`'browserIpAddress'  => explode( ',', get_post_meta( $post_id, '_customer_ip_address', true ) )[0],`

P.S.: I did hear that this plugin for WooCommerce has been abandoned.

**Comment by ChevelleNV** [comment]
I will look into your fix... as for me I did a little digging and my problem is simply an ajax request issue, the signfyd dashboard is just a iframe page that loads up the signfyd website, i removed it from the plugin...
